demo/app/controllers/select_panel_items_controller.rb

```diff
@@ -35,9 +35,10 @@ def index
     query = (params[:q] || "").downcase
 
     results = if show_results
-                SELECT_PANEL_ITEMS.select do |item|
+                res = SELECT_PANEL_ITEMS.select do |item|
                   [item[:title], item[:description]].join(" ").downcase.include?(query)
                 end
+                params.fetch(:limit_items, -1).to_i >= 0 ? res.take(params[:limit_items].to_i) : res
```

Comment:
Can we throw this in a reivew labs -- I want to make sure there are no issues with single select, limits, and searching